### DISCUSSION TITLE
How can i use pyrfc to get interval of dates

### DISCUSSION THREAD

**Discussion by zurman88** [SAP/PyRFC#323] [Q&A]
Hello friends , 

I'm trying to figure out how do i get all the records from last 4 days with Pyrfc..

Analogous to use "where" clause on Datetime column in SQL .

Thank you 🌷

**Comment by bsrdjan** [comment]
It depends on ABAP function module interface. If date selection is provided there, if can be used from PyRFC. You can check ABAP function module documentation.
